nwbwidgets/ophys.py

```diff
@@ -162,8 +162,8 @@ def __init__(self, plane_seg: PlaneSegmentation, color_wheel=color_wheel, **kwar
         self.plane_seg = plane_seg
         self.color_wheel = color_wheel
         self.progress_bar = ProgressBar()
-        self.button = widgets.Button(description='display Image Masks')
-        self.children = [self.progress_bar.container, self.button]
+        self.button = widgets.Button(description="Display ROIs")
+        self.children = [widgets.HBox([self.button, self.progress_bar.container])]
```

Comment:
Grouping then in a HBox clips the progress bar. Does that happen with you too? 
Looks like : 
![image](https://user-images.githubusercontent.com/11567561/108814041-c5802600-7577-11eb-9dee-5ea7fe2a5635.png)